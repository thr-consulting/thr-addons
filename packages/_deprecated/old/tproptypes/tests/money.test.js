import React from 'react';
import {shallow} from 'enzyme';
import Money from 'js-money';
import TPropTypes from '../src/index';

describe('Money', () => {
	const Comp = props => (
		<div>
			{props.money.amount} {props.money.currency}
		</div>
	);
	Comp.propTypes = {money: TPropTypes.money};

	it('renders without error', () => {
		expect(shallow(<Comp money={new Money(500, Money.CAD)}/>)).toMatchSnapshot();
	});
});
