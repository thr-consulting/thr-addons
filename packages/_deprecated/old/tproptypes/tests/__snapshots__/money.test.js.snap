// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Money renders without error 1`] = `
<div>
  500
   
  CAD
</div>
`;
